test/CQRS/LeanCode.CQRS.AspNetCore.Tests/CQRSObjectsRegistrationSourceTests.cs

```diff
@@ -62,6 +63,7 @@ private void AssertNotRegistered<T>()
     private static List<CQRSObjectMetadata> GetCQRSObjects()
     {
         return new CQRSObjectsRegistrationSource(
+            new ServiceCollection(),
```

Comment:
Given that the responsibility of this class changed I'd also test if handlers are properly registered, either by building this service collection and trying to resolve handlers or via a mock.